src/app/static/static-values.ts

```diff
@@ -141,6 +141,9 @@ export class StaticValues {
   ];
 
   public static strategiesListNonUniLpPools: string[] = [
+    'ETH_DAI_HODL',
```

Comment:
remove the changes from this file, please
new strategies already added on last weekends